app/graphql/mutations/update_mutation.rb

```diff
@@ -0,0 +1,31 @@
+module Mutations
+  class UpdateMutation < BaseMutation
+    field :errors, [Types::UserError], null: false
+
+    def resolve(params)
+      object = ForemanGraphqlSchema.object_from_id(params[:id], context)
+
+      object.assign_attributes(params.except(:id))
```

Comment:
Should we `authorize!` and `validate_object` before we try to assign any attributes?